server/db.js

```diff
@@ -7,7 +7,8 @@ const client = new Client({
 
 client.connect().catch(e => console.error('connection error', e.stack))
 
-const insert = (input, uid) => {
+// Old methods for cv_table
+/* const insert = (input, uid) => {
```

Comment:
Miks näitä ei vaan poisteta? Saahan ne kuitenki historiikistä jos oikein tarvii...

Reply:
En uskaltanut :D